src/features/DisplayControls/Diagnosis/components/Diagnosis.jsx

```diff
@@ -0,0 +1,139 @@
+import React, { useEffect } from "react";
+import {
+  DataTable,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableBody,
+  Table,
+  TableRow,
+  DataTableSkeleton,
+} from "carbon-components-react";
+import { useState } from "react";
+import { FormattedMessage } from "react-intl";
+import { getPatientDiagnosis } from "../utils/DiagnosisUtils";
+import PropTypes from "prop-types";
+import "../styles/Diagnosis.scss";
+
+const Diagnosis = (props) => {
+   
+    const {patientId} = props;
+    const[diagnosis, setDiagnosis] = useState([]);
+    const[isLoading, setIsLoading] = useState(true);
+
+    const NoDiagnosisMessage = (
+    <FormattedMessage
+        id={"NO_DIAGNOSIS_MESSAGE"}
+        defaultMessage={"No Diagnosis found for this patient"}
+    />
+    );
+  
+    const diagnosisHeaders = [
```

Comment:
can we move the header config to Diagnosis utility file?

Reply:
removed the file